src/Build/Construction/Solution/SolutionProjectGenerator.cs

```diff
@@ -1086,6 +1087,7 @@ private void AddProjectReference(ProjectInstance traversalProject, ProjectInstan
                 item.SetMetadata("ToolsVersion", traversalProject.ToolsVersion);
                 item.SetMetadata("SkipNonexistentProjects", "Build"); // Instruct the MSBuild task to try to build even though the file doesn't exist on disk.
                 item.SetMetadata("AdditionalProperties", GetPropertiesMetadataForProjectReference(traversalProject, SolutionConfigurationAndPlatformProperties));
+                item.SetMetadata("BuildableProject", "False");
```

Comment:
This doesn't look right to me. What does it do for metaprojects that are created for solution build order dependencies?